src/rt/cover.c

```diff
@@ -52,12 +53,14 @@ typedef struct {
 
 typedef A(cover_tag_t) tag_array_t;
 typedef A(line_range_t) range_array_t;
+typedef A(text_buf_t*) tb_array_t;
```

Comment:
This can just be an array of `char *` since you don't need to modify the strings after they are created.